spotlight-client/src/charts/utils.ts

```diff
@@ -17,12 +17,19 @@
 
 import { color } from "d3-color";
 import { interpolateRgb } from "d3-interpolate";
+import sumBy from "lodash/sumBy";
 import { useCallback, useState } from "react";
+import { STATISTIC_THRESHOLD } from "../constants";
 import { colors } from "../UiLibrary";
-import { isItemToHighlight, ItemToHighlight } from "./types";
+import { CommonDataPoint, isItemToHighlight, ItemToHighlight } from "./types";
 
 const FADE_AMOUNT = 0.45;
 
+export function isSmallData(data: CommonDataPoint[]): boolean {
```

Comment:
Checking out your latest changes, I'm no longer seeing any special treatment for small numbers. Looks like the issue is this sumBy here, we're now summing up all the values in a set of data, rather than doing what we were doing before which is `d.value < STATISTIC_THRESHOLD`.

Reply:
I pull up all latest changes and the hashed treatment still working. At least in demo mode.

Reply:
I don't see the hashed treatment working for me, where should I expect to see it? Here's a video of what I'm seeing: https://www.loom.com/share/04848c562c3849f9a523d86d1888e991

Reply:
I expect in us-nd/narratives/prison to see hashed treatment for numbers under 100

Reply:
If I change the function `isSmallData` to 
```
export function isSmallData(value: number): boolean {
  return value < STATISTIC_THRESHOLD;
}
```
and pass in `d.value` whenever we call `isSmallData` then I see this:

<img width="962" alt="Screen Shot 2022-06-08 at 5 18 40 PM" src="https://user-images.githubusercontent.com/5040739/172738463-c3001fec-bbee-4b8d-8041-3f457931862b.png">

which is what I expect.

Reply:
Oh, this was in Humphrey comment from 12 Apr:

> This was clear to me after looking at your implementation, but it would probably make more sense to apply the visual treatment to the entire visualization. For example, if the total number of events, people, etc. is less than our threshold, we would apply the treatment to all circles, bars, etc. in that visualization.

So, in order to see the treatment you should for example in `us-nd/narratives/prison/2` select `Bismark Transition Center` filter instead of `All Facilities`.

Reply:
Got it, thanks for clarifying! I missed that comment, I was relying on what was written in the PR description, could you update it? LGTM!